GOVERNANCE.md

```diff
@@ -342,6 +342,21 @@ This process kicks off once a valid nomination has been made. See ["Core Member
 8. **If the vote passes:** the nominee will be made a Core Member and all privileges will be made available to them.
 9. **If the vote fails:** the project Steward is responsible for informing the nominee with constructive, actionable feedback. (Note: this is not required if the nomination was made in the #core channel, or if the nominee was otherwise not made aware of their nomination).
 
+## Voting: TSC Member Nomination
+
+This process kicks off once a valid nomination has been made. See ["TSC Member - Nomination Process"](#nomination-4) above for more details on nomination.
+
+**Who can vote:** All Core members (L3 and above) except the nominee.
+
+1. A vote thread should be created in Discord `#core` channel (the private channel for Core members).
```

Comment:
```suggestion
1. A vote thread should be created in Discord `#core` channel (the private channel for Core members).
```
Again, I *think* all TSC members are core, and therefore in a case like this, is it worth noting that we'd I think create a *private* thread that excludes the nominee?